src/components/PartialDeposit.tsx

```
@@ -1,54 +1,64 @@
+					</button>
+				</legend>
+				<input
+					type="number"
+					placeholder="Type here"
+					className="input input-primary input-sm w-full"
+					name="amount"
+					max={formatEther(balance.balance)}
+					value={formatEther(amount)}
+					onChange={(e) => {
+						if (e.target.value === '') { setAmount(0n); return; }
+						try { setAmount(parseEther(e.target.value)); } catch { }
```

ESLint

[error] Empty block statement.

(no-empty)